### ISSUE TITLE
An Introduction to Dylan - Objects documentation needs refinement.

### ISSUE THREAD

**Issue by enlight** [dylan-lang/opendylan#601]
http://opendylan.org/documentation/intro-dylan/objects.html

The Built-In Classes section reads "A few of the most important classes are shown here:" but the following paragraph doesn't correspond to this line.

**Comment by fraya** [comment]
Documentation in this section seems consistent.

**Comment by cgay** [comment]
Agreed. Maybe there was a problem displaying the table at the time enlight reported this or something. Let's close this.
